.github/workflows/sync_s3_cache.yml

```diff
@@ -0,0 +1,28 @@
+# This workflow uploads third-party dependencies to OSSCI S3.
+# For more information and documentation, see:
+# * https://github.com/pytorch/builder/main/thirdparty_deps/README.md
+# * https://github.com/pytorch/builder/pull/1096
+# * .github/pytorch-cached-s3-dependencies.yml
+
+name: Update S3 dependencies on public OSSCI S3 buckets
+on:
+  schedule:
+    # Sync every hour
+    - cron: "*/60 * * * *"
+  # Have the ability to trigger this job manually using the API as well
+  workflow_dispatch:
+
+jobs:
+  sync-s3-cache:
+    runs-on: ubuntu-18.04
+    if: ${{ github.repository == 'pytorch/pytorch' }}
+    steps:
+      - name: checkout
+        uses: actions/checkout@v3
+      - name: Sync S3 file cache
+        env:
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_S3_UPDATE_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_S3_UPDATE_SECRET_ACCESS_KEY }}
+        uses: docker://pytorch/sync_s3_thirdparty_deps
```

Comment:
TIL: so this will automatically pull and run the docker image and run it with the argument?  I'm curious how it works. Do you have any docs from github at hand? :)

Reply:
You can check some info about [dockerized actions here](https://docs.github.com/en/actions/creating-actions/creating-a-docker-container-action).

Regarding the call with the argument, here is how the magic happens:
1. `ENTRYPOINT` in the `Dockerfile` points [to the script to invoke](https://github.com/pytorch/builder/pull/1096/files#diff-9f8654449913609b1e56a2c48ea773f28ff1cd2627dc8ddd3667e0992a782ae2R6)
2. [Dockerized action](https://docs.github.com/en/actions/creating-actions/creating-a-docker-container-action) checks out the docker image, and runs it [mounting the working directory there](https://github.com/orgs/community/discussions/25637#discussioncomment-3248560). This way the config file is available from inside the docker container running the step.